### PR TITLE
Add Course Playlists to free-courses-hi.md

### PR DESCRIPTION
**Pull request by MalevolentStrix**
## What does this PR do?
Add 17 courses to the Hindi Courses section for C++, DevOps, Java, ReactJS, PHP as well as Django
Also added extra information to an ambiguous set of two courses in which one is a playlist and another is not

### Why is this valuable (or not)?

This will help beginners to find useful and relevant courses for learning these languages in one place instead of going through all the trash.

### How do we know it's really free?

All of them are youtube playlists which are obviously free and also even I have followed some of these tutorials.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -117,10 +127,13 @@
 
 #### React
 
-* [React Js Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - CodeWithHarry
+* [React JS Tutorial in Hindi | React JS for Beginner to Advanced | Step by Step Video Tutorials](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdnIQKgnTeR1T9-1ltJEaJh) - WsCubeTech
```
Comment (eshellman) [inline courses/free-courses-hi.md:130]: escape the |s

**Review by eshellman — CHANGES_REQUESTED** [review]
please double check each item. Thanks!

File: courses/free-courses-hi.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-hi.md:130]:
```suggestion
* [React JS Tutorial in Hindi \| React JS for Beginner to Advanced \| Step by Step Video Tutorials](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdnIQKgnTeR1T9-1ltJEaJh) - WsCubeTech
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -117,10 +127,13 @@
...
+* [React Js Tutorials in Hindi(Playlist)](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - Haris Ali Khan, CodeWithHarry
```
[the remark this answers is not in the document]
Comment (davorpa) [reply courses/free-courses-hi.md:131]:
```suggestion
* [React Js Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - Haris Ali Khan, CodeWithHarry
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -95,9 +100,14 @@
 
 ### Java 
 
+* [Complete Java Programming in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtnqhXTJwd-oqVRwO3bLZCGV) - Anand Kumar, CodeitUp
+* [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Geeky Shows
 * [Java + DS + Algorithms](https://www.youtube.com/playlist?list=PLKKfKV1b9e8ps6dD3QA5KFfHdiWj9cB1s) - Apni Kaksha (Anuj)
 * [Java + DSA](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3LtFWcvwpqTkUSlB32kJop) - Apna College
 * [Java Foundation Course \| Hindi](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFj7YSPl2ulcpwy-mwj1SSk) - Pepcoding
+* [Java Programming Tutorial(HINDI/URUDU)](https://www.youtube.com/playlist?list=PLiOa6ike4WAHljIOitb3vR0nXQgneUedR) - Vikas Pandey, Easytuts4you
```
[the remark this answers is not in the document]
Comment (davorpa) [reply courses/free-courses-hi.md:108]:
```suggestion
* [Java Programming Tutorial (HINDI/URDU)](https://www.youtube.com/playlist?list=PLiOa6ike4WAHljIOitb3vR0nXQgneUedR) - Vikas Pandey, Easytuts4you
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -54,6 +54,10 @@
 
 * [C++ Full Course | C++ Tutorial \| Data Structures & Algorithms](https://www.youtube.com/playlist?list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ) - Apna College
 * [C++ Programming in Hindi](https://www.youtube.com/playlist?list=PLDA2q3s0-n15yszaZ2yRKEoxY-WWkuAt4) - Sumit Bisht (Edutainment 1.0)
+* [C++ Programming in Hindi](https://www.youtube.com/playlist?list=PLbGui_ZYuhijXuOfBSdQgK296Y7wUDWLn) - Geeky Shows
```
Comment (davorpa) [inline courses/free-courses-hi.md:57]: Add real instructor name

```suggestion
* [C++ Programming in Hindi](https://www.youtube.com/playlist?list=PLbGui_ZYuhijXuOfBSdQgK296Y7wUDWLn) - Rajesh Kumar, Geeky Shows
```

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -54,6 +54,10 @@
...
+* [C++ Programming Tutorial in Hindi](https://www.youtube.com/playlist?list=PLoVVmGDgrrnS5_TiSg193ezTPd-Ukb25k) - Rakesh Roshan, Learn TechToTech
+* [C++ Tutorial For Begineers In Hindi](https://www.youtube.com/playlist?list=PLnSDvcENZlwA6YDSfoieM1bl-Y3ALcnL5) - Abhishek Shrivastava, Micro Solution
+* [C++ Tutorial for Beginners | C++ Tutorials In Hindi](https://www.youtube.com/playlist?list=PLmGElG-9wxc8VMy1nNHDQldH2dU8Y08s7) - Manish Gehlot, WsCube Tech Programming Concepts
```
Comment (davorpa) [inline courses/free-courses-hi.md:60]: Escape pipes

```suggestion
* [C++ Tutorial for Beginners \| C++ Tutorials In Hindi](https://www.youtube.com/playlist?list=PLmGElG-9wxc8VMy1nNHDQldH2dU8Y08s7) - Manish Gehlot, WsCube Tech Programming Concepts
```

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -78,6 +82,7 @@
 ### DevOps
 
 * [DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
+* [DevOps Tutorials in Hindi/Urdu | Complete Devops including Git,Jenkins,Maven,Chef,docker,Ansible and Kubernetes](https://www.youtube.com/playlist?list=PLBGx66SQNZ8aPsFDwb79JrS2KQBTIZo10) - Bhupinder Rajput, Technical Guftgu
```
Comment (davorpa) [inline courses/free-courses-hi.md:85]: Complete title + escape pipes

```suggestion
* [DevOps Tutorials in Hindi/Urdu \| Devops सीखने का सबसे आसान तरीका \| Complete Devops including Git,Jenkins,Maven,Chef,docker,Ansible and Kubernetes](https://www.youtube.com/playlist?list=PLBGx66SQNZ8aPsFDwb79JrS2KQBTIZo10) - Bhupinder Rajput, Technical Guftgu
```

File: courses/free-courses-hi.md [hunk 7]
```diff
@@ -95,9 +100,14 @@
 
 ### Java 
 
+* [Complete Java Programming in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtnqhXTJwd-oqVRwO3bLZCGV) - Anand Kumar, CodeitUp
+* [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Geeky Shows
```
Comment (davorpa) [inline courses/free-courses-hi.md:104]: Add real instructor name

```suggestion
* [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Rajesh Kumar, Geeky Shows
```

File: courses/free-courses-hi.md [hunk 8]
```diff
@@ -117,10 +127,13 @@
 
 #### React
 
-* [React Js Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - CodeWithHarry
+* [React JS Tutorial in Hindi | React JS for Beginner to Advanced | Step by Step Video Tutorials](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdnIQKgnTeR1T9-1ltJEaJh) - WsCubeTech
+* [React Js Tutorials in Hindi(Playlist)](https://youtube.com/playlist?list=PLu0W_9lII9agx66oZnT6IyhcMIbUMNMdt) - Haris Ali Khan, CodeWithHarry
+* [React JS(Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Geeky Shows
```
Comment (davorpa) [inline courses/free-courses-hi.md:132]: Complete with instructor real name + exact title name

```suggestion
* [React JS (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Rajesh Kumar, Geeky Shows
```

File: courses/free-courses-hi.md [hunk 9]
```diff
@@ -132,6 +145,8 @@
 
 ### PHP
 
+* [Core PHP(Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigFdLdbSI2EM2MrJB7I0j-B) - Geeky Shows
```
Comment (davorpa) [inline courses/free-courses-hi.md:148]: Complete with instructor real name + exact title

```suggestion
* [Core PHP (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigFdLdbSI2EM2MrJB7I0j-B) - Rajesh Kumar, Geeky Shows
```

File: courses/free-courses-hi.md [hunk 10]
```diff
@@ -132,6 +145,8 @@
...
+* [PHP Tutorial in Hindi/Urudu](https://www.youtube.com/playlist?list=PL0b6OzIxLPbyrzCMJOFzLnf_-_5E_dkzs) - Yahoo Baba
```
Comment (davorpa) [inline courses/free-courses-hi.md:149]: Fix title typos

```suggestion
* [PHP Tutorial in Hindi / Urdu](https://www.youtube.com/playlist?list=PL0b6OzIxLPbyrzCMJOFzLnf_-_5E_dkzs) - Yahoo Baba
```

File: courses/free-courses-hi.md [hunk 11]
```diff
@@ -146,6 +161,8 @@
 
 #### Django
 
+* [Django Tutorial for Beginners(Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92yWUMgLpWrXtegKxrWLRnRv) - Vijay Manral, CS Geeks
+* [Django(Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigchy8DTw4pX4duTTpvqlh6) - GeekyShows
```
Comment (davorpa) [inline courses/free-courses-hi.md:165]: Complete with instructor real name + exact title

```suggestion
* [Django(Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigchy8DTw4pX4duTTpvqlh6) - Rajesh Kumar, GeekyShows
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Maybe some resource needs to be reordered after.

No squash is needed while PR is iterated

**Comment by MalevolentStrix** [comment]
I have applied the changes as mentioned in the review :slightly_smiling_face:

**Review by davorpa — APPROVED** [review]
Well done. It's enough to me. Let's wait for Eric decision

**Comment by MalevolentStrix** [comment]
> Well done. It's enough to me. Let's wait for Eric decision

Sure, Thank you for spending time to review my PR

**Comment by MalevolentStrix** [comment]
Hacktoberfest is going to end in about 3 hours according my country's time  :cry:. 
I don't know whether it is following the local time or not.
Can I get the label for now? I will make any changes to the pr if it is required.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**